### ISSUE TITLE
Plop skips validation if the argument is given in the CLI

### ISSUE THREAD

**Issue by GeeWee** [amwmedia/plop#94]
Hi. I've noticed that plop skips inquirer validation if the prompt is given in the CLI.
Take the following example. This should always fail

```
plop.setGenerator('component', {
		description: 'this generates a new common component',
		prompts: [
			{
				type: 'input',
				name: 'name',
				message: 'the name of the component. (PascalCased)',
				validate: () => 'nope',
			},
		],
		actions: [
			() => {
				return 'yay';
			}
		],
	});
```
![image](https://user-images.githubusercontent.com/6381792/31554680-9a69d490-b03e-11e7-88ac-ff0908f93b2e.png)

As it does.

However when using the shorthand it skips the validation:

![image](https://user-images.githubusercontent.com/6381792/31554738-cbae841a-b03e-11e7-8dba-b8346b17043a.png)

It would be nice if it did the validation no matter which way the arguments were entered.

**Comment by amwmedia** [comment]
agreed, this is something that we should definitely add. Thanks for calling it out!

**Comment by amwmedia** [comment]
fixed in v2.0
